app/src/main/java/com/sopt/now/compose/presentation/main/MainActivity.kt

```diff
@@ -119,7 +120,7 @@ fun showMain(userData: SignUpData?) {
                 }
 
                 2 -> {
-                    MyPageScreen(userData)
+                    MyPageScreen(memberId)
```

Comment:
```suggestion
2 ->  MyPageScreen(memberId)
```
과 같이 한줄로 표기하는게 가독성 면에서 좋아보여요